.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/vscode/devcontainers/base:dev-bookworm
```

Comment:
Are the Dockerfile changes here intentional? Doesn't appear immediately relevant this change. If this is necessary, can this be moved to a separate commit?